_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -1,159 +1,356 @@
+
+## Background
+
+- [G124: Adding links at the top of the page to each area of the content](https://www.w3.org/WAI/WCAG21/Techniques/general/G124)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+A list of links to different sections of content each having a focusable semantic link element and when activated moves the focus to the associated section of content.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation">Skip to navigation</a></li>
+			<li><a href="#search">Skip to search</a></li>
+			<li><a href="#main">Skip to main content</a></li>
+		</ul>
+		<!-- Repeated Content -->
+		<nav id="navigation">Navigation section</nav>
+		<form role="search" id="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+The list of links are visible when focused.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+		<style>
+			#skiplinks a {
+				position: absolute;
+				top: 100px;
+			}
+			#skiplinks a:focus {
+				top: 0px;
+			}
+		</style>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation">Skip to navigation</a></li>
+			<li><a href="#search">Skip to search</a></li>
+			<li><a href="#main">Skip to main content</a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form role="search" id="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+The list of links when activated move focus to heading elements.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#firstheading">First heading</a></li>
+			<li><a href="#secondheading">Second heading</a></li>
+		</ul>
+		<!-- Repeated Content -->
+		<h2 id="firstheading">First heading section</h2>
+		<!-- Content -->
+		<h2 id="secondheading">Second heading section</h2>
+		<!-- Content -->
+	</body>
+</html>
+```
+
+#### Passed Example 4
+
+The list of links use `aria-label` to provide an accessible name.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation" aria-label="Skip to navigation"></a></li>
+			<li><a href="#search" aria-label="Skip to search"></a></li>
+			<li><a href="#main" aria-label="Skip to main content"></a></li>
+		</ul>
+		<!-- Repeated content -->
+		<nav id="navigation">Navigation section</nav>
+		<form role="search" id="search">Search section</form>
+		<main id="main">Main content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 5
+
+The list of links use `aria-describedby` to provide an accessible name.
```

Comment:
I think that `aria-describedby` is poorly supported and we generally prefer using `aria-labelledby` in examples.

Reply:
removed this example!